CONTRIBUTING.md

```diff
@@ -0,0 +1,101 @@
+# Contributing to c2cgeoportal
+
+Thanks for your interest in contributing to a GeoMapFish project.
+
+## Asking Questions
+
+Please ask questions about using the library on [Stack Overflow using the tag 'openlayers']
+(http://stackoverflow.com/questions/tagged/openlayers).
+
+When you want to get involved and discuss new features or changes, please use [the mailing list]
+(https://groups.google.com/forum/#!forum/openlayers-dev).
```

Comment:
Why do we mention the openlayers-dev forum here? Copy-paste mistake?

Reply:
To be discussed, @ybolognini